src/external_command/rustup_command.rs

```diff
@@ -121,16 +121,16 @@ impl RustupCommand {
 
         Ok(RustupOutput {
             output,
-            stdout: once_cell::sync::OnceCell::new(),
-            stderr: once_cell::sync::OnceCell::new(),
+            stdout: std::cell::OnceCell::new(),
```

Comment:
Please use OnceLock or LazyLock instead, since we also used once_cell::sync before.